scripts/offline_eval/export_requests.py

```diff
@@ -48,10 +52,15 @@ def fetch_client():
         else:
             raise ValueError(f"Invalid organization: {organization}")
 
-    def process_request(request: Request):
+    def process_request(request: Request, include_scenario_name: bool = False):
         raw_request: typing.Dict = fetch_client().convert_to_raw_request(request)
+
         # Only export requests that are not in the cache
         if not store.contains(raw_request):
+            if include_scenario_name:
+                # When `include_scenario_name` is True, include the scenario name in the request
+                raw_request["scenario"] = scenario_name
```

Comment:
I think it's better if we are strict about every JSON schema corresponding to some dataclass...  that way, it's less error-prone.